[2024-03-28T05:41:23.999+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_get_tmd_api.get_data_api scheduled__2024-03-27T21:15:00+00:00 [queued]>
[2024-03-28T05:41:24.030+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_get_tmd_api.get_data_api scheduled__2024-03-27T21:15:00+00:00 [queued]>
[2024-03-28T05:41:24.031+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-28T05:41:24.080+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): get_data_api> on 2024-03-27 21:15:00+00:00
[2024-03-28T05:41:24.094+0000] {standard_task_runner.py:60} INFO - Started process 138130 to run task
[2024-03-28T05:41:24.112+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'test_get_tmd_api', 'get_data_api', 'scheduled__2024-03-27T21:15:00+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/test_get_tmd_api.py', '--cfg-path', '/tmp/tmpu4y_paqy']
[2024-03-28T05:41:24.119+0000] {standard_task_runner.py:88} INFO - Job 134: Subtask get_data_api
[2024-03-28T05:41:24.279+0000] {task_command.py:423} INFO - Running <TaskInstance: test_get_tmd_api.get_data_api scheduled__2024-03-27T21:15:00+00:00 [running]> on host c5a6ac367c49
[2024-03-28T05:41:24.569+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_get_tmd_api' AIRFLOW_CTX_TASK_ID='get_data_api' AIRFLOW_CTX_EXECUTION_DATE='2024-03-27T21:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-27T21:15:00+00:00'
[2024-03-28T05:41:25.861+0000] {test_get_tmd_api.py:43} INFO - **** filename : data_2024_03_28_1200.json
[2024-03-28T05:41:25.896+0000] {test_get_tmd_api.py:49} INFO - JSON data saved to /opt/***/dags/data_get.json
[2024-03-28T05:41:25.897+0000] {test_get_tmd_api.py:53} INFO - local file path : /opt/***/dags/data_get.json
[2024-03-28T05:41:25.898+0000] {test_get_tmd_api.py:57} INFO - bucket name : ds525-capstone-test-49
[2024-03-28T05:41:25.901+0000] {test_get_tmd_api.py:60} INFO - dst : data_test/data_2024_03_28_1200.json
[2024-03-28T05:41:25.972+0000] {connection.py:269} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-03-28T05:41:25.989+0000] {base.py:83} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-03-28T05:41:26.910+0000] {gcs.py:557} INFO - File /opt/***/dags/data_get.json uploaded to data_test/data_2024_03_28_1200.json in ds525-capstone-test-49 bucket
[2024-03-28T05:41:26.918+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-28T05:41:26.944+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=test_get_tmd_api, task_id=get_data_api, execution_date=20240327T211500, start_date=20240328T054124, end_date=20240328T054126
[2024-03-28T05:41:27.023+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-28T05:41:27.069+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
